test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/MessagePackHubProtocolTests.cs

```diff
@@ -228,6 +228,16 @@ public static IEnumerable<object[]> TestDataNames
                 message: AddHeaders(TestHeaders, new CancelInvocationMessage("xyz")),
                 binary: "kwWDo0Zvb6NCYXKyS2V5V2l0aApOZXcNCkxpbmVzq1N0aWxsIFdvcmtzsVZhbHVlV2l0aE5ld0xpbmVzsEFsc28KV29ya3MNCkZpbmWjeHl6"),
 
+            // StreamComplete Messages
```

Comment:
Same here as in JSON, add a test for an invocation message that has a `StreamPlaceholder` parameter